src/llmcompressor/pipelines/cache.py

```
@@ -13,6 +14,35 @@
+
+    @functools.wraps(func)
+    def wrapper(self, *args, **kwargs):
+        try:
+            return func(self, *args, **kwargs)
+        except Exception as exception:
+            cache_size = self.size()
+            size_info = ", ".join(
+                f"{device}: {nbytes / (1024**2):.2f} MB"
+                for device, nbytes in cache_size.items()
+            )
+            error_msg = f"{str(exception)}\nIntermediatesCache size: {size_info}"
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)